Applications/DataExplorer/DataView/StationTreeView.cpp

```diff
@@ -237,38 +230,28 @@ void StationTreeView::showDiagramPrefsDialog()
 
 void StationTreeView::writeStratigraphiesAsImages(QString listName)
 {
-	std::map<std::string,
-	         GeoLib::Color*> colorLookupTable =
-	        static_cast<VtkStationSource*>(static_cast<StationTreeModel*>(model())->vtkSource(
-	                                               listName.
-	                                               toStdString()))
-	        ->getColorLookupTable();
+	std::map<std::string, GeoLib::Color> colorLookupTable =
+		static_cast<VtkStationSource*>(static_cast<StationTreeModel*>
+			(model())->vtkSource(listName.toStdString()))->getColorLookupTable();
 	std::vector<ModelTreeItem*> lists = static_cast<StationTreeModel*>(model())->getLists();
 	std::size_t nLists = lists.size();
 	for (std::size_t i = 0; i < nLists; i++)
-		if ( listName.toStdString().compare( lists[i]->data(0).toString().toStdString() )
-		     == 0 )
-		{
-			const std::vector<GeoLib::Point*>* stations =
-			        dynamic_cast<BaseItem*>(lists[i]->getItem())->getStations();
+	{
+		if ( listName.toStdString().compare( lists[i]->data(0).toString().toStdString() ) != 0 )
```

Comment:
isn't it possible to compare QString directly without converting to std::string?